<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite skipped="0" hostname="DESKTOP-84CJOHJ" name="org.testNGEx3.SoftAssertLoginLogoutActitimeEx2" tests="1" failures="1" timestamp="2022-08-15T16:16:51 IST" time="14.056" errors="0">
  <testcase name="testMethod" time="14.056" classname="org.testNGEx3.SoftAssertLoginLogoutActitimeEx2">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	Dashboard title is changed expected [actiTIME - Enter Time-Track11] but found [actiTIME - Enter Time-Track],
	Dashboard url is changed expected [http://localhost:90/user/submit_tt.do11] but found [http://localhost:90/user/submit_tt.do]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Dashboard title is changed expected [actiTIME - Enter Time-Track11] but found [actiTIME - Enter Time-Track],
Dashboard url is changed expected [http://localhost:90/user/submit_tt.do11] but found [http://localhost:90/user/submit_tt.do]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:47)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:31)
at org.testNGEx3.SoftAssertLoginLogoutActitimeEx2.testMethod(SoftAssertLoginLogoutActitimeEx2.java:32)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- testMethod -->
  <system-out>
    <![CDATA[Entered Username.
Entered Password.
Click on login button.
Dashborad Title is as Expected!!
Dashborad URL is as Expected!!]]>
  </system-out>
</testsuite> <!-- org.testNGEx3.SoftAssertLoginLogoutActitimeEx2 -->
